kadas/app/milx/kadasmilxintegration.cpp

```diff
@@ -412,6 +390,127 @@ void KadasMilxIntegration::saveMilxly()
   delete zip;
 }
 
+void KadasMilxIntegration::exportKml()
+{
+  kApp->unsetMapTool();
+
+  KadasMilxExportDialog exportDialog;
+  exportDialog.setExportMode( KadasMilxExportDialog::ExportMode::Kml );
+
+  if ( exportDialog.exec() == QDialog::Rejected )
+  {
+    return;
+  }
+
+  QStringList exportLayers = exportDialog.selectedLayers();
```

Comment:
```suggestion
  const QStringList exportLayers = exportDialog.selectedLayers();
```

Avoids a detach in the loop IIRC